mmcv/transforms/wrappers.py

```diff
@@ -0,0 +1,307 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+from collections.abc import Sequence
+from typing import Any, Callable, Dict, List, Optional, Union
+
+import numpy as np
+
+import mmcv
+from .base import BaseTransform
+from .builder import TRANSFORMS
+from .utils import cache_random_params
+
+# Indicator for required but missing keys in results
+NotInResults = object()
+
+# Import nullcontext if python>=3.7, otherwise use a simple alternative
+# implementation.
+try:
+    from contextlib import nullcontext
+except ImportError:
+    from contextlib import contextmanager
+
+    @contextmanager
+    def nullcontext(resource=None):
+        try:
+            yield resource
+        finally:
+            pass
+
+
+@TRANSFORMS.register_module()
+class Compose(BaseTransform):
+    """Compose a data pipeline with a sequence of transforms.
+
+    Args:
+        transforms (list[dict | callable]): Either config
+          dicts of transforms or transform objects.
+    """
```

Comment:
Example should also be added.

Reply:
I think the example of `Compose` may not be necessary considering it's hardly used by users directly.

Reply:
Yes, but it helps developers to understand the usages of this class.